src/components/UsersTableItem/UserTableItem.spec.tsx

```diff
@@ -1183,25 +1186,6 @@ describe('Testing User Table Item', () => {
       resetAndRefetch: resetAndRefetchMock,
     };
 
-    const mocks = [
-      {
-        request: {
-          query: REMOVE_MEMBER_MUTATION,
-          variables: {
-            userId: '123',
-            orgId: 'xyz',
-          },
-        },
```

Comment:
Why was this removed?

Reply:
declared but value was never read in the code
<img width="841" alt="image" src="https://github.com/user-attachments/assets/9312a7f2-f057-4589-b41d-77c7916e4da1" />